test/unit/app/models/editionable_worldwide_organisation_test.rb

```diff
@@ -70,8 +88,8 @@ class EditionableWorldwideOrganisationTest < ActiveSupport::TestCase
     worldwide_organisation.roles << create(:ambassador_role, :vacant)
     worldwide_organisation.roles << create(:deputy_head_of_mission_role, :vacant)
     worldwide_organisation.roles << create(:worldwide_office_staff_role, :vacant)
-
     assert_nil worldwide_organisation.primary_role
+
```

Comment:
Minor nitpick: this line break probably wasn't intended to be moved.

Reply:
Apologies, this will be a result of rebasing 400 times!